src/components/shared/Bandeau/Bandeau.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+
+import { ReactElement, useContext } from 'react'
+
+import styles from './Bandeau.module.css'
+import { clientContext } from '../ClientContext'
+
+export interface BandeauInformations {
+  titre: string | undefined
+  description: string | undefined
+}
+
+export default function Bandeau(): ReactElement | null {
+  const { bandeauInformations } = useContext(clientContext)
+
+  const isDisplayed = (): boolean => {
+    return bandeauInformations?.titre !== undefined && bandeauInformations.description !== undefined
+  }
+
+  return isDisplayed() ? (
```

Comment:
```suggestion
  return bandeauInformations ? (
```

Toute cette logique de vérification de l'existence d'un bandeau devrait se faire depuis le composant parent. Autrement, on a ici un composant potentiellement vide.